lib/cli/commands/closechannel.ts

```diff
@@ -28,9 +28,15 @@ export const builder = (argv: Argv) => argv
     type: 'number',
     description: 'for Connext only - the amount to extract from the channel',
   })
+  .option('fee', {
+    type: 'number',
+    description: 'the manual fee rate set in sat/byte that should be used when crafting the closure transaction in the channel',
+    default: 0,
+  })
   .example('$0 closechannel BTC 028599d05b18c0c3f8028915a17d603416f7276c822b6b2d20e71a3502bd0f9e0b', 'close BTC channels by node key')
   .example('$0 closechannel BTC CheeseMonkey', 'close BTC channels by alias')
   .example('$0 closechannel BTC CheeseMonkey --force', 'force close BTC channels by alias')
+  .example('$0 closechannel BTC CheeseMonkey 25', 'close BTC channels by alias with 25 sat/byte fee')
```

Comment:
This command doesn't work, because the order of the args the 25 is assigned to the `force` variable. I'd make this command `$0 closechannel BTC CheeseMonkey --fee 25`.

Reply:
Ah true that